Gems/Terrain/Code/Source/EditorComponents/EditorTerrainSystemComponent.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/Component.h>
+
+#include <AzToolsFramework/Entity/EditorEntityContextBus.h>
+
+namespace Terrain
+{
+    /// System component for Terrain editor
+    class EditorTerrainSystemComponent
+        : public AZ::Component
+        , private AzToolsFramework::EditorEvents::Bus::Handler
+    {
+    public:
+        AZ_COMPONENT(EditorTerrainSystemComponent, "{5e9f2200-9099-4325-babd-6a533a1abea8}");
```

Comment:
nit: pretty much everywhere else the letters are in CAPs

Reply:
Changed!